crates/ruff_python_formatter/resources/test/fixtures/ruff/statement/match.py

```diff
@@ -141,3 +141,24 @@ def foo():
     no_comments
     ):
         pass
+
+match foo:
```

Comment:
can you add `(["a", "b"])` as a test case?

Reply:
Added!